tests/appsec/iast/sink/test_hardcoded_secrets.py

```diff
@@ -30,6 +30,22 @@ def test_hardcoded_secrets_exec(self):
         vuln = hardcode_secrets[0]
         assert vuln["location"]["path"] == self._get_expectation(self.location_map)
 
+    # rules from the extended set for secret detection need to know the variable name in addition to the literal value
+    @missing_feature(library="golang", reason="Not implemented yet")
+    @missing_feature(library="java", reason="Not implemented yet")
+    @missing_feature(library="dotnet", reason="Not implemented yet")
+    @missing_feature(library="python", reason="Not implemented yet")
+    @missing_feature(library="ruby", reason="Not implemented yet")
+    @missing_feature(library="php", reason="Not implemented yet")
+    @missing_feature(library="cpp", reason="Not implemented yet")
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Not implemented yet")
```

Comment:
I think that you want to say something like:
`@released(nodejs='5.11.0')`
But anyway, instead of doing this, couldn't we create new Test class with the new test? It would be the right way to set the release version and missing features in the yaml, where they are expected.

Reply:
I mean, something like:

```
class Test_HardcodedSecrets_Extended:
```

Reply:
ok, at the moment `test_hardcoded_secrets_extended_exec` is using the results from the previous request (`test_hardcoded_secrets_exec`). If we create a new test class, I guess you'd create a new endpoint to test it instead of reusing the endpoint from `test_hardcoded_secrets_exec`

Reply:
yes, but not necessarily, you can call to the same endpoint in other test, but I think that it could be cleaner if we add new endpoint